### ISSUE TITLE
 Error : Push.Configure may only be called once

### ISSUE THREAD

**Issue by verdavaine** [raix/push#125]
I get the error  'Push.Configure may only be called once'
when using the Push.Configure function on client.
unless it is called inside Meteor.startup

But I want the users be able to configure the notifications while using the app and decide if they want to hear a sound or a vibration. 
So it is necessary to be able to call Push.Configure outside Meteor.startup

**Comment by jpgilchrist** [comment]
@verdavaine I don't think this is really an issue with this package. This is more of a core understanding of how notifications work. ios/android support different payloads (json structure) for their push notifications. The values in the payload have meanings and specifically for android you can send in the notification itself badge, sound, etc. I would try toggling those in each notification instead of calling Push.Configure multiple times. 

See the documentation here: https://github.com/raix/push/blob/v3.x/docs/ADVANCED.md#overwriting-platform-specific-values

Hope that helps

**Comment by verdavaine** [comment]
Yes Thank u for your help !

2015-10-17 19:52 GMT+02:00 James Gilchrist <notifications@github.com>:
